Intrusion.tproj

```diff
@@ -628,7 +632,7 @@
               </ProjectNodeViewModel>
             </SubNodes>
           </ProjectNodeViewModel>
-          <ProjectNodeViewModel xsi:type="ProjectFileNodeViewModel" Name="InitCommon.sqx" IsExpanded="false" FileName="Common\InitCommon.sqx">
```

Comment:
This is a dilemma with TypeSqf. To keep clean from these kinds of lines, always close TypeSqf when doing commit. Or as an alternative select the root node of the projekt, right click and select collapse all sub nodes.